Web/src/pages/ForgotPassword.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import {
+  ChakraProvider,
+  Input,
+  Box,
+  VStack,
+  Text,
+  Button,
+  useToast
+} from "@chakra-ui/react";
+import theme from "../theme.js";
+import logo from "../assets/examplelogo.png";
+import { Link, useNavigate } from 'react-router-dom';
+import supabase from "../supabaseClient.js";
+
+function ForgotPassword() {
+  const [email, setEmail] = useState("");
+  const [loading, setLoading] = useState(false);
+  const toast = useToast();
+  const navigate = useNavigate();
+  
+
+  const handleSendResetLink = async () => {
+    setLoading(true);
+
+    const { error } = await supabase.auth.resetPasswordForEmail(email);
+
+    if (error) {
+      toast({
+        title: 'Error',
+        description: error.message,
+        status: 'error',
+        duration: 5000,
+        isClosable: true,
+      });
```

Comment:
idk if you have checked responsiveness but as a note just make sure to make sure this looks decent on mobile view as well. you can access this on any browser tool using right click and the inspect section